Ignite links

nirmitparikh8 opened the pull request:
I made 4 main ignite event pages. Added Buttons on the main pages going to the different pages of the event. Updated Latest Works. Removed YC finder.

Dhvani35729 commented on functions/index.js:
```diff
@@ -32,31 +32,58 @@ app.get("/workshops", function(request, response) {
 app.get("/fuse19", function(request, response) {
   response.render("pages/fuse19", {
-    name: "fuse"
+    name: "fuse 2019"
```
Why is there a space here? I think the convention has been no spaces.

nirmitparikh8 replied:
wdym by convention?

Dhvani35729 replied:
Convention meaning what does the existing code do

nirmitparikh8 replied:
IDK why there is a space. How does that impact the code though?

KrishKrosh replied:
It is better to use the same conventions throughout the code to reduce confusion

```suggestion
    name: "fuse19"
```

nirmitparikh8 replied:
Okie, I will change that

Dhvani35729 commented:
> _No description provided._

Please provide a description of changes made

nirmitparikh8 commented:
>I fixed the pictures, latest work blocks, removed fuse 2019 from media, removed stempower 2019 event, added fuse 2019 to latest works, and fixed the margins on STEMpower 2019. Couldn't fix JUnto Support and Ignite on the home page. I don't know how to center it. Can someone help? I committed a couple of changes recently, tell me if you got them.

Dhvani35729 commented on functions/views/pages/stempower.ejs:
```diff
@@ -0,0 +1,116 @@
+					<div class="display-t">
+						<div class="display-tc">
+							<h1 class="animate-box" data-animate-effect="fadeInUp">STEMpower</h1>
+							<h2 class="animate-box" data-animate-effect="fadeInUp">STEMpower is a branch of YouthComputing that holds mini presentations at elemntary schools. This branch showcases and exposes cool technologies, exposing the tech world to the younger students.</h2>
```
Spelling error. Says: "at elemntary schools"

Dhvani35729 commented on functions/views/pages/fuse.ejs:
```diff
@@ -0,0 +1,89 @@
+								<h1></h1>
+
+						<div class="col-md-12">
+							<h3>Drones at Fuse 2017</h3>
```
This div is too close to the top one. Add a margin top

Dhvani35729 commented on functions/views/pages/fuse19.ejs:
```diff
@@ -15,50 +15,66 @@
-				<h3>Pictures from the event: </h3>
-				<a href="https://woodbuffalo.snapd.com/events/view/1104996">https://woodbuffalo.snapd.com/events/view/1104996</a>
-				<a href="https://www.dropbox.com/sh/94n5qcy9r6pcxum/AAAH7fc36wikvLBEzGBa4FdAa?dl=0">https://www.dropbox.com/sh/94n5qcy9r6pcxum/AAAH7fc36wikvLBEzGBa4FdAa?dl=0</a>
+					<p>YouthComputing is hosting Fuse 2019 at Westwood Community High School on December 21. Fuse is an oppurtunity for youth to showcase various technological inventions and it also promotes science, technology, engineering, art and mathematics.</p>
```
change is hosting to hosted

Dhvani35729 commented on functions/views/pages/fuse19.ejs:
```diff
@@ -15,50 +15,66 @@
+					</div>
+					<p></p>
+					<p></p>
+					<p> <a href="https://drive.google.com/drive/folders/13lqEoBKIoQVqhJMRkmJdQJf0vyRAT4l3?usp=sharing" class="round">More Pictures From Fuse 2019</a></p>
```
this link is to old drive, update the link

nirmitparikh8 replied:
Admin needs to enable link sharing out of the YC organization

nirmitparikh8 replied:
Talked with Krish and we cannot share outside of our organization, so I just kept the old link

Dhvani35729 commented on functions/views/pages/fuse17.ejs:
```diff
@@ -16,52 +16,67 @@
+			<h1></h1>
+			<div class="col-md-12">
+				<h2>Fuse 2017</h2>
+				<p>YouthComputing is hosting Fuse 2017 at</p> 
```
change is hosted to hosted

Dhvani35729 commented on functions/views/pages/ccd.ejs:
```diff
@@ -16,788 +16,71 @@
 							<h1 class="animate-box" data-animate-effect="fadeInUp">Community Code Day</h1>
-							<h2 class="animate-box" data-animate-effect="fadeInUp">2018 | Arts & Culture</h2>
-							<a href="https://communitycodeday.com/"><h2>https://communitycodeday.com/</h2></a>
+							<h2 class="animate-box" data-animate-effect="fadeInUp">Community Code Day is Fort McMurray's first community technology conference. People can participate in a hackathon, and / or they can choose from a selection of speakers and workshops to learn how to use technology as a beneficial factor in their endeavours.</h2>
```
change first to annual

Dhvani35729 commented:
@nirmitparikh8 , you should add the pictures from CryptoChicks workshop to workshops page

nirmitparikh8 commented:
> @nirmitparikh8 , you should add the pictures from CryptoChicks workshop to workshops page

I can't find the pictures for the cryptochicks workshop

nirmitparikh8 commented:
> @nirmitparikh8 , you should add the pictures from CryptoChicks workshop to workshops page

Done

Dhvani35729 commented:
+1
